app/assets/javascripts/controllers/container_dashboard/container_dashboard_utils.js

```diff
@@ -0,0 +1,116 @@
+angular.module('containerDashboard').factory('ContainerDashboardUtils', [function ContainerDashboardUtils () {
+  var createProvidersStatus = function() {
+    return {
+      title: "Providers",
+      count: 0,
+      href: "../ems_container/show_list",
+      notifications: []
```

Comment:
The aggregate status card with the tall layout uses the field is notifications rather than notification since it allows multiples. The providers card is showing multiple provider types via the notifications field.